extensions/wikia/UserLogin/js/UserLoginFacebook.js

```diff
@@ -83,20 +83,24 @@
 						UserLoginModal.$modal.trigger('close');
 					}
 				});
+			this.bucky.timer.stop('loginSetup');
```

Comment:
I moved this up here so that we will only be tracking the setup code, not the code that's actually login the user in. This will affect the graphs but will give us a more accurate reading.